From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Liav A <liavalb@gmail.com>
Date: Sat, 25 Jun 2022 05:45:29 +0300
Subject: [PATCH] Use global program_name variable from gnu dir

Without this patch being applied, there would be a conflict between the
variables between the 2 locations, so it will not compile otherwise.

---
 src/global.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/global.c b/src/global.c
index fb3abe9..9ea63aa 100644
--- a/src/global.c
+++ b/src/global.c
@@ -20,6 +20,7 @@
 #include <system.h>
 
 #include <sys/types.h>
+#include "gnu/progname.h"
 #include "cpiohdr.h"
 #include "dstring.h"
 #include "extern.h"
@@ -184,9 +185,6 @@ unsigned int warn_option = 0;
 /* Extract to standard output? */
 bool to_stdout_option = false;
 
-/* The name this program was run with.  */
-char *program_name;
-
 /* A pointer to either lstat or stat, depending on whether
    dereferencing of symlinks is done for input files.  */
 int (*xstat) ();
